tests/Modifier/AbstractPriceModifierTest.php

```diff
@@ -1,7 +1,10 @@
 <?php
+namespace Blesta\Pricing\Tests\Unit\Modifier;
```

Comment:
Namespace should be `Blesta\Pricing\Tests\Modifier;` *OR* test should be moved into a `Unit` directory within `tests/` (notice case).